elasticassignment/ageofempires/management/commands/index_civilizations.py

```diff
@@ -0,0 +1,45 @@
+from django.core.management.base import BaseCommand
```

Comment:
Evaluate your design (of all `Command` classes) on SOLID principles: https://anuragdhadse.com/solid-principles/
Does it follow any of its principles?